pootle/settings/50-project.conf

```diff
@@ -52,6 +50,7 @@ TEMPLATES = [
                 'pootle_misc.context_processors.pootle_context',
             ],
             'loaders': [
+                # Put here to accept template customization
```

Comment:
im not sure if this is necessary - this is for loaders not for paths - so its confusing